components/inquiry_questionnaire/map.coffee

```diff
@@ -1,4 +1,13 @@
+{ FORCED_LOGIN_INQUIRY } = require('sharify').data
+
+determineSteps = ->
+  if FORCED_LOGIN_INQUIRY is 'force_login'
```

Comment:
Rather than make conditionals at each step of the process, we just make a new map for the questionnaire to follow if the user lands in the test bucket.